src/sentry/web/forms/accounts.py

```diff
@@ -163,6 +163,16 @@ class RegistrationForm(forms.ModelForm):
         widget=forms.TextInput(attrs={'placeholder': 'you@example.com'}))
     password = forms.CharField(
         widget=forms.PasswordInput(attrs={'placeholder': 'something super secret'}))
+    subscribe = forms.BooleanField(
+        label=_('Subscribe to product updates newsletter'),
```

Comment:
I'm going to hold off on this fwiw, since we're not going to be shipping this in OSS yet. I'll back out these minor things. I wouldn't want to call it "monthly" and I don't want to lock us into something here until we're ready to handle how we want to communicate with onpremise, and it sounds liek that's not fully planned out yet. So I'm in the middle of yanking these things out with just the minimal so we can support it on sentry.io first.